src/java.base/share/classes/java/io/RandomAccessFile.java

```diff
@@ -206,8 +206,8 @@ public RandomAccessFile(String name, String mode)
      *             not denote an existing regular file, or if the mode begins
      *             with {@code "rw"} but the given file object does not denote
      *             an existing, writable regular file and a new regular file of
-     *             that name cannot be created, or if some other error occurs
-     *             while opening or creating the file
+     *             that pathname cannot be created, or if some other error
+     *             occurs while opening or creating the file
      * @throws      SecurityException  if a security manager exists and its
      *             {@code checkRead} method denies read access to the file
      *             or the mode is {@code "rw"} and the security manager's
```

Comment:
GitHub UI isn't allowing me to add a review comment on line 195, so adding it here. Should line 195 be changed to say:

```
also called with the pathname of the {@code file} argument to see if write access to the file...
```
to match what it says a few lines above for the `checkRead` security check?

Reply:
You are correct - thanks. So changed in ef91651b35cd30a7a11ac75ebbdc03a9c29c59e3.